Core/include/Acts/Vertexing/GaussianTrackDensity.hpp

```diff
@@ -125,7 +133,7 @@ class GaussianTrackDensity {
   ///
   /// @return z position of the global maximum
   Result<std::optional<double>> globalMaximum(
-      State& state, const std::vector<InputTrack>& trackList) const;
+      State& state, const std::vector<InputTrack>& trackList,double time) const;
```

Comment:
Same as above - I think we need a different return type and `time` is part of the input tracks